fe/fe-core/src/main/java/com/starrocks/catalog/FileTable.java

```diff
@@ -104,7 +102,13 @@ public Map<String, String> getFileProperties() {
     }
 
     public List<RemoteFileDesc> getFileDescs() throws DdlException {
-        Configuration configuration = new Configuration();
+        Configuration configuration;
+        CloudCredential cloudCredential = CloudCredentialFactory.buildStorageCloudCredential(fileProperties);
+        if (cloudCredential != null) {
+            configuration = cloudCredential.getConfiguration();
+        } else {
+            configuration = new Configuration();
```

Comment:
if we set all `properties` from create stmt to this configuration, we can support consutom hadoop configuration